### ISSUE TITLE
How to check the recombination event?

### ISSUE THREAD

**Issue by DuttaAnik** [lh3/psmc#45]
Hello,
Thank you for this super useful tool. I was able to run it successfully. However, I have a question regarding choosing the optimum `-t `value and `-p` model. I was reading in the manual where it is stated that "the `-p` and `-t` options are manually chosen such that after 20 rounds of iterations, at least ~10 recombinations are inferred to occur in the intervals each parameter spans."

My question is, how can I check whether ~10 recombinations are inferred or not? I checked the output file after psmc run but could not find it. Maybe I am looking at the wrong place.

It would really help me to choose the correct -p model if you can tell me where to find that information. 
Thank you.

**Comment by DuttaAnik** [comment]
Hello, can someone give me an answer to this question above? It really matters to me at the moment.
Thank you.

**Comment by dareen22** [comment]
This is very helpful, thank you @Buffan3369 for sharing!

**Comment by DuttaAnik** [comment]
Indeed, this is awesome. Thanks a lot, @Buffan3369. It worked finally!

**Comment by rapamycin** [comment]
@Buffan3369 Thank you so much.

Does it means that π_K/(ρ*C_π) >20, or **π_K > 20*ρ*C_π**?

**Comment by rapamycin** [comment]
Does it means that π_K/(ρ*C_π) >20, or π_K > 20*ρ*C_π?

**Comment by rapamycin** [comment]
Thank you so much ! @Buffan3369 
Does it means that π_K/(ρ*C_π) should larger than 20, or π_K should larger than 20*ρ*C_π?
